sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/persons/PersonDirectorySteps.java

```diff
@@ -18,33 +18,40 @@
 
 package org.sormas.e2etests.steps.web.application.persons;
 
-import static org.sormas.e2etests.pages.application.persons.EditPersonPage.UUID_INPUT;
-import static org.sormas.e2etests.pages.application.persons.EditPersonPage.getByPersonUuid;
+import static org.sormas.e2etests.pages.application.cases.CaseDirectoryPage.*;
+import static org.sormas.e2etests.pages.application.persons.EditPersonPage.*;
 import static org.sormas.e2etests.pages.application.persons.PersonDirectoryPage.*;
 
+import com.google.common.truth.Truth;
 import cucumber.api.java8.En;
+import java.util.concurrent.TimeUnit;
 import javax.inject.Inject;
 import javax.inject.Named;
 import org.openqa.selenium.By;
+import org.sormas.e2etests.common.DataOperations;
+import org.sormas.e2etests.enums.CommunityValues;
+import org.sormas.e2etests.enums.DistrictsValues;
+import org.sormas.e2etests.enums.PresentCondition;
+import org.sormas.e2etests.enums.RegionsValues;
+import org.sormas.e2etests.helpers.AssertHelpers;
 import org.sormas.e2etests.helpers.WebDriverHelpers;
-import org.sormas.e2etests.pojo.web.Person;
 import org.sormas.e2etests.state.ApiState;
 import org.sormas.e2etests.steps.web.application.contacts.EditContactPersonSteps;
 import org.sormas.e2etests.steps.web.application.events.EditEventSteps;
+import org.testng.Assert;
 
 public class PersonDirectorySteps implements En {
   private final WebDriverHelpers webDriverHelpers;
-  protected Person createdPerson;
 
   @Inject
   public PersonDirectorySteps(
       WebDriverHelpers webDriverHelpers,
       @Named("ENVIRONMENT_URL") String environmentUrl,
-      ApiState apiState) {
+      ApiState apiState,
+      AssertHelpers assertHelpers,
+      DataOperations dataOperations) {
     this.webDriverHelpers = webDriverHelpers;
 
-    // TODO refactor all BDD methods naming to be more explicit regarding where data comes from
```

Comment:
don't remove TODOs, they are added there to keep a list with items which should be addressed